server/service/IndividualServicesService.js

```diff
@@ -519,8 +495,17 @@ function getAllApplicationList() {
         let tcpClientUuid = serverLtp[0];
         let applicationAddress = await tcpClientInterface.getRemoteAddressAsync(tcpClientUuid);
         let applicationPort = await tcpClientInterface.getRemotePortAsync(tcpClientUuid);
-        let clientApplication = new clientApplicationInformation(applicationName, applicationReleaseNumber, applicationAddress, applicationPort);
-        clientApplicationList.push(clientApplication);
+        let applicationProtocal = await tcpClientInterface.getRemoteProtocolAsync(tcpClientUuid)
+        listInfoApplication = {
+          "application-name": applicationName,
```

Comment:
Please remove the hardcoded values of application-name , release-number, protocol, port